### DISCUSSION TITLE
Newbie!

### DISCUSSION THREAD

**Discussion by dlandi** [oqtane/oqtane.framework#4354] [General] [answered]
Hello @sbwalker 
I am finally taking a deep dive into Oqtane, and I am impressed. 
I've gone through your seven-part "Oqtane Module Development" series on YouTube, but it is four years old.
I am now watching the "On .NET Live—Exploring Oqtane for Blazor and .NET MAUI" video.

What other resources should I be looking at?

Can you please send me an invitation to your Discord?

Thanks in advance.

Dennis Landi

**Comment by leigh-pointer** [comment]
@dlandi here is a link. It will be nice to see you there https://discord.gg/BTcyTfE7

**Comment by dlandi** [comment, marked as answer]
@leigh-pointer That is a link to the MubBlazor forum. Thanks.

I was specifically asking for an invite to the Oqtane discord forum. Does one exist, or is this GitHub board the only community channel?

**Reply by leigh-pointer** [reply]
Sorry about that https://discord.gg/cB53V8GX

**Reply by dlandi** [reply]
Thanks!

**Reply by alignwaivers** [reply]
Looks like that expired, I would love to join there as well if I may! I've been learning oqtane slowly, as was pointed out there aren't a ton of resources yet but I'd love to help with that if I could :)

**Reply by leigh-pointer** [reply]
Try this https://discord.gg/BnPny88avK
